e2e-tests/insurance/cypress/e2e/journeys/account/create/confirm-email/confirm-email-verification-expired-in-inactive-set.spec.js

```diff
@@ -0,0 +1,85 @@
+import { INSURANCE_ROUTES as ROUTES } from '../../../../../../../constants/routes/insurance';
+import api from '../../../../../../../commands/api';
+
+const {
+  START,
+  ACCOUNT: {
+    CREATE: { CONFIRM_EMAIL, VERIFY_EMAIL, CONFIRM_EMAIL_RESENT },
+  },
+} = ROUTES;
+
+const baseUrl = Cypress.config('baseUrl');
+
+context(
+  'Insurance - Account - Create - Confirm email page - expired token and isInactive flag set - As an Exporter I want to verify my email address for account creation, So that I can activate my email address and use it to create a digital service account with UKEF',
```

Comment:
constant for `isInactive`?

Can we rename the file to have "state" instead of "set"?